client/landing/jetpack-cloud/components/log-item/index.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React, { ReactNode } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import CardHeading from 'components/card-heading';
+
+import './style.scss';
+
+interface Props {
+	children?: ReactNode;
+	header: string;
+	subheader: string;
+	highlight?: string;
+	tag?: string;
+}
+
+class LogItem extends React.PureComponent< Props > {
+	static defaultProps = {
+		highlight: 'info',
+		tag: 'new',
+	};
+
+	renderHeader() {
+		const { header, subheader, tag } = this.props;
+
+		return (
+			<div className="log-item__header-wrapper">
+				<span className="log-item__tag">{ tag }</span>
```

Comment:
Should we be displaying the tag only if `tag` is specified?

Reply:
Resolved in baae8bd44670bdcfd50bfb32ce3d97d0cb622e50.